hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -106,9 +107,9 @@ public SCMBlockProtocolServer(OzoneConfiguration conf,
       StorageContainerManager scm) throws IOException {
     this.scm = scm;
     this.conf = conf;
-    final int handlerCount =
+    final int handlerCount = conf.getInt(OZONE_SCM_BLOCK_HANDLER_COUNT_KEY,
         conf.getInt(OZONE_SCM_HANDLER_COUNT_KEY,
-            OZONE_SCM_HANDLER_COUNT_DEFAULT);
+            OZONE_SCM_HANDLER_COUNT_DEFAULT));
```

Comment:
Let's add a utility method in `OzoneConfiguration`.
```java
//OzoneConfiguration.java
  public int getInt(String name, String fallbackName, int defaultValue, Consumer<String> log) {
    String value = this.getTrimmed(name);
    if (value == null) {
      value = this.getTrimmed(fallbackName);
      if (log != null) {
        log.accept(name + " is not set.  Fallback to " + fallbackName + ", which is set to " + value);
      }
    }
    if (value == null) {
      return defaultValue;
    }
    return Integer.parseInt(value);
  }
```
Then, we can use it here and also for the other cases.
```java
    final int handlerCount = conf.getInt(OZONE_SCM_BLOCK_HANDLER_COUNT_KEY,
        OZONE_SCM_HANDLER_COUNT_KEY, OZONE_SCM_HANDLER_COUNT_DEFAULT, LOG::info);
```